pkg/sass_api/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
-## 16.0.3-dev
+## 16.0.3
 
-* No user-visible changes.
+* Add the missing members of `RecursiveAstVisitor`.
```

Comment:
I think this may be confusing, as in I didn't understand immediately what "missing members" meant. Can you clarify this?